Bookish.Web/Models/CatalogueViewModel.cs

```diff
@@ -1,15 +1,34 @@
+using System;
 using System.Collections.Generic;
+using System.Linq;
 using Bookish.DataAccess;
 
 namespace Bookish.Web.Models
 {
     public class CatalogueViewModel
     {
         public IEnumerable<CatalogueEntry> Catalogue { get; }
+        public string Filter { get; }
+        public int Page { get; }
+        public int LastPage { get; }
 
-        public CatalogueViewModel(IEnumerable<CatalogueEntry> catalogue)
+        public CatalogueViewModel(IEnumerable<CatalogueEntry> catalogue, int page = 1, string filter = "")
         {
-            Catalogue = catalogue;
+            Page = page;
+            Filter = filter;
+
+            Catalogue = catalogue.ToList();
+            LastPage = (int)Math.Ceiling(((decimal) Catalogue.Count() / 5));
+            Catalogue = Catalogue
+                .Skip((page - 1) * 5)
+                .Take(5);
+        }
+
+        public string GetListClasses(int pageNumber)
```

Comment:
I find it a bit hard to tell what this is for based on the name. I assume it's to style the page numbers? If so then maybe something like `GetPageNumberClass` would be more descriptive? 🙂